com.onepassword.OnePassword.yml

```diff
@@ -0,0 +1,54 @@
+app-id: org.onepassword.onepassword
```

Comment:
`onepassword.org` doesn't look like the right domain. `1password.com` is the one.

Reply:
this mean the app-id doesn't meet the requirements.

Reply:
this doesn't even match the appstream file.